RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 Any new bugs, requests, or missing features should be reported as [Github issues](https://github.com/NordicSemiconductor/zcbor/issues).
 
 ## Improvements:
+ * requirements.txt: Change PyYAML requirement from >6.0.0 to >5.4.1 to be more widely compatible.
```

Comment:
Is there a point to refer to what you are trying to be compatible with or is it generally just more common to use an older pyyaml version?